src/app/speechbar/speechbar.component.html

```diff
@@ -19,12 +19,12 @@
     <button class="stackedButton" mat-raised-button [appInteractionEventHandler]="buttonRow.speak.handler" [disabled]="speaking"
             [class.scanHighlight]="buttonRow.speak.isHighlighted()"
             [class.scanSelect]="buttonRow.speak.isSelected()"
-            style="--button-image: url('/assets/images/speechbubble.png')"
+            style="--button-image: url('assets/images/speechbubble.png')"
```

Comment:
This is a bug I found once I'd deployed to staging. Thought I'd fix it while I was here!